src/main/java/com/dynatrace/loadrunner/CommandLineParser.java

```diff
@@ -0,0 +1,56 @@
+package com.dynatrace.loadrunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class CommandLineParser {
+
+	final public Map<String, String> arguments;
+
+	public CommandLineParser(String[] args) {
+
+		arguments = new HashMap<>();
+
+		for (int i = 0; i < args.length; i++) {
+			String key = null;
+			if (args[i].startsWith("-")) {
+				key = args[i].substring(1);
+				if ((i + 1) < args.length) {
+					if (args[i + 1].charAt(0) != '-') {
+						arguments.put(key, args[i + 1]);
+						i++;
```

Comment:
modifying iterator inside a loop is an unusual approach - should be at least documented:
```
i++; // skip next argument
```
or using an iterator:
```
if(iterator.hasNext()) {
  arguments.put(key, iterator.next());
}
```
(no need to skip element, since `next()` already does that and shifts iterator position)